nengo/builder/ensemble.py

```diff
@@ -95,6 +96,14 @@ def get_gain_bias(ens, rng=np.random):
         max_rates = get_samples(ens.max_rates, ens.n_neurons, rng=rng)
         intercepts = get_samples(ens.intercepts, ens.n_neurons, rng=rng)
         gain, bias = ens.neuron_type.gain_bias(max_rates, intercepts)
+        if gain is not None and (
+                not np.all(np.isfinite(gain)) or np.any(gain <= 0.)):
+            raise BuildError(
```

Comment:
Would it be useful to link to a discussion of this problem? Maybe a forum post with the delightful doodles you made?